pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1003,11 +1013,22 @@ BEGIN
                 timescaledb.compress_segmentby = 'series_id',
                 timescaledb.compress_orderby = 'time, value'
             ); $$, metric_table_name);
+
         --chunks where the end time is before now()-1 hour will be compressed
-        PERFORM add_compress_chunks_policy(format('SCHEMA_DATA.%I', metric_table_name), INTERVAL '1 hour');
+        IF new_compress_policy THEN
```

Comment:
Nit: I think this is more clear. But I'll leave up to you.
```suggestion
        IF SCHEMA_CATALOG.get_timescale_major_version() >= 2 THEN
```